src/main.js

```diff
@@ -48,6 +55,27 @@ angular.module(MODULE_NAME, [])
       },
 
       link(scope, element, attrs, ngModel) {
+        const _setViewValue = () => {
+          let viewValue;
+          switch (_dateFormat) {
```

Comment:
I know, but changing the switch expression the way you're describing will cause the `_setViewValue()` (or `getViewValue()`) function to return before it reaches line 75: `ngModel.$setViewValue(viewValue);`, which will break the function.